src/test/kotlin/uk/gov/justice/digital/hmpps/courtregister/resource/CourtResourceTest.kt

```diff
@@ -29,6 +29,9 @@ class CourtResourceTest : IntegrationTest() {
   @MockBean
   private lateinit var courtRepository: CourtRepository
 
+  @MockBean
```

Comment:
We've found that it's often better to use `@SpyBean` instead but push it up to `IntegrationTest`... this allows you to use the bean as a mock if you so wish but if not then it just proxies through to the real bean.  The main benefit is that Spring doesn't need a different context for every combination of MockBean and real bean - all tests use a SpyBean so they can all run in the same Spring context.